pipeline/trusted-resources/pkg/trustedtask/trustedtaskrun.go

```diff
@@ -136,8 +131,27 @@ func prepareTask(t v1beta1.TaskObject) (v1beta1.Task, []byte, error) {
 		APIVersion: "tekton.dev/v1beta1",
 		Kind:       "Task"}
 	task.Spec = t.TaskSpec()
-	task.ObjectMeta = t.TaskMetadata()
 
+	// exclude the fields populated by system.
```

Comment:
Probably worth adding a test for this func - in particular to make sure we're covering cases like empty/missing data (e.g. metadata, maps, etc.)